runtime/protocol/http-client-engines/http-client-engine-crt/jvm/src/aws/smithy/kotlin/runtime/http/engine/crt/CrtHttpEngineConfig.kt

```diff
@@ -44,13 +45,48 @@ public class CrtHttpEngineConfig private constructor(builder: Builder) : HttpCli
      */
     public var clientBootstrap: ClientBootstrap? = builder.clientBootstrap
 
+    /**
+     * Certificate Authority content in PEM format
+     * Mutually exclusive with caFile and caDir.
+     */
+    public var caRoot: String? = builder.caRoot
+
+    /**
+     * Path to the root certificate. Must be in PEM format.
+     * Mutually exclusive with caRoot.
+     */
+    public var caFile: String? = builder.caFile
+
+    /**
+     * Path to the local trust store. Can be null.
+     * Mutually exclusive with caRoot.
+     */
+    public var caDir: String? = builder.caDir
+
+    /**
+     * TLS cipher suite preference for connections.
+     * Controls which cipher suites are available during TLS negotiation.
+     */
+    public var cipherPreference: TlsCipherPreference = builder.cipherPreference
+
+    /**
+     * Whether to verify the peer's certificate during TLS handshake.
+     * When false, accepts any certificate (insecure, for testing only).
```

Comment:
docs: I would remove what's in parenthesis. We shouldn't prescribe what's secure or not, or when to use `verifyPeer`